servers/src/main/java/tachyon/worker/block/allocator/AllocatorFactory.java

```diff
@@ -35,8 +35,10 @@ public static Allocator create(AllocatorType allocatorType, BlockMetadataManager
         return new GreedyAllocator(metaManager);
       case MAX_FREE:
         return new MaxFreeAllocator(metaManager);
+      case ROUND_ROBIN:
+        return new RoundRobinAllocator(metaManager);
       default:
-        return new GreedyAllocator(metaManager);
+        return new MaxFreeAllocator(metaManager);
```

Comment:
Oh I see, let's keep the greedy version for now since thats the default in master.